app/views/authors/show.html.erb

```diff
@@ -0,0 +1,41 @@
+<h1 class="text-center m-4 h1">Author: <%= @author.first_name + " " + @author.last_name %></h1>
+
+<table class="table w-50 table-bordered table-striped table-hover m-auto text-center">
+  <tr>
+    <th>Id</th>
+    <th>Name</th>
+    <th>Date Of Birth</th>
+    <th>Email</th>
+  </tr>
+  <tr>
+    <td><%= @author.id %></td>
+    <td><%= @author.first_name + " " + @author.last_name %></td>
+    <td><%= @author.date_of_birth %></td>
+    <td><%= @author.email %></td>
+  </tr>
+
+</table>
+
+<h3 class="text-center m-4"> <%= @author.first_name + " " + @author.last_name %> has <%= @books.count %> books.</h3>
+
+<table class="table w-50 table-bordered table-striped table-hover m-auto text-center">
+  <tr>
+    <th>Id</th>
+    <th>Name</th>
+    <th>Price</th>
+    <th>Author Id</th>
+  </tr>
+  <% @books.each do |book| %>
+    <tr>
+      <td><%= link_to book.id, book %></td>
```

Comment:
Any reason to add a `link` here?

Reply:
I added link here if anyone wants to check details about some specific book of the author.